CODE_OF_CONDUCT.md

```diff
@@ -4,7 +4,7 @@ As contributors and maintainers of this project, we pledge to respect all people
 
 We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
 
-Examples of unacceptable behavior by participants include the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+Examples of unacceptable behavior by participants include the use of sexual language or imagery, derogatory comments or personal attacks, trolling, public or private harassment including but not limited to publishing other people's private information or electronic or physical address, insults, or other unprofessional conduct.
```

Comment:
Sadly, the list is getting a little unwieldy at this length. How about we bullet it?

```markdown
Examples of unacceptable behavior by participants include:

+ the use of sexual language or imagery
+ derogatory comments or personal attacks
+ behavior intended only to disrupt (trolling)
+ public or private harassment
+ dissemination of other people's private information or electronic or physical address
+ insults or other unprofessional conduct
```